swan-lake/development-tutorials/source-code-dependencies/package-references.md

```diff
@@ -346,7 +346,19 @@ warning: Detected conflicting jar files:
 
 **Define the scope for a dependency**
 
-By default, the scope takes the value `default` which will add it to the final executable JAR file. If you want to restrict a certain platform dependency to be used only for testing, specify the scope as `testOnly`. This will add the platform dependency to the test runtime but will avoid packing it into the final executable JAR file.
+By default, when the scope has not been explicitly specified for a platform dependency in the `Ballerina.toml`, it will be packaged into the final executable JAR or the BALA. There are two scopes that can be used to restrict this behaviour.
+
+***'TestOnly' scope***
+
+If you want to restrict a certain platform dependency to be used only for testing, specify the scope as `testOnly`. This will add the platform dependency to the test runtime but will avoid packing it into the final executable JAR file.
+
+***'provided' scope***
+
+If you want to restrict a certain platform dependency from been packed into the BALA, specify the scope as `provided`. This will add the platform dependency to the final executable JAR but not to the BALA.
```

Comment:
shall we use the terms `jar file` and `bala file` instead of `JAR` and `BALA`?

Reply:
Addressed in #8836